src/TesApi.Web/Management/ContainerRegistryProvider.cs

```diff
@@ -67,14 +70,32 @@ public virtual async Task<ContainerRegistryInfo> GetContainerRegistryInfoAsync(s
             }
 
             return await LookUpAndAddToCacheContainerRegistryInfoAsync(imageName);
+        }
```

Comment:
AWS hosted public registry? etc. This will fail to work once the public brings their workflows to this.